cmd/finch-credhelper/windows-creds.go

```diff
@@ -0,0 +1,57 @@
+//go:build windows
```

Comment:
Typically, platform specific files are suffixed with the platform. For example, this one would be `creds_windows.go`, and a mac file would be `creds_darwin.go`. There are some examples of this in the code base